tests/gdb-tests/tests/test_windbg.py

```diff
@@ -23,7 +23,7 @@ def test_windbg_dX_commands(start_binary):
         cmd = cmd_prefix + " nonexistentsymbol"
         assert gdb.execute(cmd, to_string=True) == (
             "usage: XX [-h] address [count]\n"
-            "XX: error: argument address: invalid HexOrAddressExpr value: 'nonexistentsymbol'\n"
```

Comment:
@disconnect3d it's interesting that this change resulted in a different exception being raised here. Do you know if the new exception shown is the correct one, or is something broken and I should figure out how to get the old one back?

Reply:
That's weird: https://github.com/pwndbg/pwndbg/blob/301d0b9ccdccf5e59480a5f5b9a2e80ed5292b57/pwndbg/commands/__init__.py#L566-L586C23